nixos/doc/manual/development/what-happens-during-a-system-switch.chapter.md

```diff
@@ -1,50 +1,50 @@
 # What happens during a system switch? {#sec-switching-systems}
 
-Running `nixos-rebuild switch` is one of the more common tasks under NixOS.
-This chapter explains some of the internals of this command to make it simpler
-for new module developers to configure their units correctly and to make it
-easier to understand what is happening and why for curious administrators.
+Running `nixos-rebuild switch` is one of the more common tasks in NixOS.
+This chapter explains some of the internals of this command to make it easier
+for new module developers to configure their units correctly, and to make it
+easier for curious administrators to understand what is happening and why.
 
 `nixos-rebuild`, like many deployment solutions, calls `switch-to-configuration`
-which resides in a NixOS system at `$out/bin/switch-to-configuration`. The
-script is called with the action that is to be performed like `switch`, `test`,
-`boot`. There is also the `dry-activate` action which does not really perform
-the actions but rather prints what it would do if you called it with `test`.
-This feature can be used to check what service states would be changed if the
-configuration was switched to.
+which is located at `$out/bin/switch-to-configuration` in a NixOS system. The
+script is called with the action that is to be performed, such as `switch`, `test`,
+`boot`. There is also the `dry-activate` action which does not actually perform
+the actions but rather prints out what it would do if you called it with `test`.
+This feature can be used to check chich service states would be changed if the
+configuration were switched to.
 
-If the action is `switch` or `boot`, the bootloader is updated first so the
+If the action is `switch` or `boot`, the boot loader will be updated first, so the
```

Comment:
Unrelated to this PR, as this is only rewording an existing text, but I think `switch-to-configuration boot` only installs a boot loader, but does *not* make the configuration the active one on the next boot: https://github.com/NixOS/nixpkgs/issues/82851